SPECS/metal3-baremetal-operator/metal3-baremetal-operator.spec

```diff
@@ -0,0 +1,41 @@
+Summary:        Bare Metal Operator implements a Kubernetes API for managing bare metal hosts
+Name:           metal3-baremetal-operator
+Version:        0.4.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/metal3-io/baremetal-operator
+# Created using ./generate-sources.sh
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  golang >= 1.14
+
+%description
+The Bare Metal Operator implements a Kubernetes API for managing bare metal
+hosts.  It maintains an inventory of available hosts as instances of the
+BareMetalHost Custom Resource Definition. The Bare Metal Operator knows how to
+inspect the host's hardware details and report them on the corresponding
+BareMetalHost, provision hosts with a desired image, and clean a host's disk
+contents before or after provisioning.
+
+%prep
+%autosetup
+
```

Comment:
Can you add a `%check` section with whatever tests they have in their makefile (equivalent of `go test`)